subworkflows/local/amp.nf

```diff
@@ -36,7 +36,7 @@ workflow AMP {
 
     // AMPIR
     if ( !params.amp_skip_ampir ) {
-        AMPIR ( ch_faa_for_ampir, params.amp_ampir_model, params.amp_ampir_minlength, params.amp_ampir_minprobability )
+        AMPIR ( ch_faa_for_ampir, params.amp_ampir_model, params.amp_ampir_minlength, 0.0 )
```

Comment:
uhhh just want to double check: @jasmezz is there a reason why this value is a required input channel rather than via `args`? (or do we not do `args` for R package)

Reply:
Since it is a feature of the module to filter the predictions, I put it in as required. Params with default value would also be okay, if that looks nicer? For the pipeline setting to 0 makes sense in any case as @Darcy220606 explained :+1:

Reply:
Ok I just went and looked at the module and I see what you mean, indeed R case is different becuse you can't insert anything in args across multiple places (and there isn't necessarily a 1:1 ration of args, args2, args3 etc.). So nevermind, we are good now! Just need AMPcombi in and can add the global probability filter then